credebl/studio
# feat: copy functionality for ids and disabled rolewise button if no access

## What?
- Implemented copy functionality for IDs for the below pages:
    - Issuance
    - Verification
    - View Schema
    - Verify popup

- Disabled role-wise buttons when the user does not have access to use it

## Why?
To improve user experience
